ecs-agent/netlib/data/network_client.go

```diff
@@ -14,23 +14,61 @@
 package data
 
 import (
+	"crypto/rand"
 	"encoding/json"
+	"math/big"
+	"strconv"
 
 	"github.com/aws/amazon-ecs-agent/ecs-agent/data"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/metrics"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/netlib/model/tasknetworkconfig"
 
+	"github.com/pkg/errors"
 	bolt "go.etcd.io/bbolt"
 )
 
 const (
 	networkNamespaceBucketName = "networkNamespaceConfiguration"
+	portBucketName             = "port"
+
+	// geneveDstPortMin and geneveDstPortMax specifies the range from which the GENEVE destination port will be selected.
+	// The range is currently set from 6081 to 6151 which gives us 70 ports per baremetal host to access the same subnet.
```

Comment:
I thought baremetal host refers to the devices that we use for FoF. Isn't this change focusing on WP instances?

Reply:
We are importing code for firecracker platform also into the shared library to have all the network related code in one place. But we have not made any modification to FoF workflow.